Dockerfile

```diff
@@ -2,28 +2,23 @@ FROM node:8
 
 LABEL maintainer="yeongjinnn@gmail.com"
 
-# Install latest chrome dev package.
+# Install latest chrome package.
 # Note: this installs the necessary libs to make the bundled version of Chromium that Pupppeteer
 # installs, work.
 RUN apt-get update && apt-get install -y wget --no-install-recommends \
     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
     && apt-get update \
-    && apt-get install -y google-chrome-unstable \
+    && apt-get install -y google-chrome-stable \
       --no-install-recommends \
     && rm -rf /var/lib/apt/lists/* \
     && apt-get purge --auto-remove -y curl \
     && rm -rf /src/*.deb
 
-# Uncomment to skip the chromium download when installing puppeteer. If you do,
-# you'll need to launch puppeteer with:
-#     browser.launch({executablePath: 'google-chrome-unstable'})
-# ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
```

Comment:
I'd recommend documenting that a user will be able to use this environment variable when they are running docker-compose. Ultimately it probably should be left out of the docker build scripts.